### PR TITLE
Support ZXHN F450(EPON ONU)

### PR DESCRIPTION
**Pull request by gh-orange**
Tested and working

### CODE REVIEW

File: zcu/zte.py [hunk 1]
```diff
@@ -12,7 +12,10 @@ def read_header(infile):
     header_magic = struct.unpack('>4I', infile.read(16))
     if header_magic == constants.ZTE_MAGIC:
         # 128 byte header
-        header = struct.unpack('>28I', infile.read(112))
+        raw_header = infile.read(112)
```
Comment (mkst) [inline zcu/zte.py:15]: I can't tell what value this is adding?

File: zcu/zte.py [hunk 2]
```diff
@@ -77,6 +80,20 @@ def add_header(payload, signature, payload_type, version):
                                                      0, 0, 0, 0,
                                                      0, 0, 0, 0,
                                                      0, 0, 0, 0)))
+    if signature == b'ZXHN F450(EPON ONU)':
```
Comment (mkst) [inline zcu/zte.py:83]: we shouldn't be adding product-specific code like this, instead the options that write this 'style' should be exposed,

**Comment by mkst** [comment]
Thanks for your interest in this project - your PR will need some work before we can merge though - let me know if you need any help.

File: zcu/zte.py (hunk 2 above)
Comment (gh-orange) [reply zcu/zte.py:83]: > we shouldn't be adding product-specific code like this, instead the options that write this 'style' should be exposed,

The reason is that the device 'F450 (EPON)' export configuration has Little-Endian numbers, and the keygen is like an enhanced 'serial_keygen' method to output the result (key + serialA, iv + serialB)!

**Comment by mkst** [comment]
Can you try this branch https://github.com/mkst/zte-config-utility/tree/zxhn-f450-support ? (or provide a config.bin so that i can test it out myself?)

**Pull request closed without merging**